tool_test.go

```diff
@@ -0,0 +1,165 @@
+package codacytool
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+)
+
+func setup() {
+	os.Setenv(_basePathEnvVar, testsResourcesLocation)
```

Comment:
I don't think is a good idea to rely on env variables set by us

Reply:
You're right, removed it ✅